packages/scripts/webapp/preset/config/webpack.config.dev.js

```diff
@@ -1,26 +1,16 @@
 const path = require('path');
 
-module.exports = ({ getUserConfig }) => {
-	const plugins = [];
-	if (getUserConfig('cmf')) {
-		// eslint-disable-next-line global-require,import/newline-after-import
-		const ReactCMFWebpackPlugin = require('@talend/react-cmf-webpack-plugin');
-		plugins.push(new ReactCMFWebpackPlugin({ watch: true }));
-	}
-
-	return {
-		mode: 'development',
-		watchOptions: {
-			aggregateTimeout: 300,
-			poll: 1000,
-		},
-		devServer: {
-			port: 3000,
-			stats: 'errors-only',
-			historyApiFallback: true,
-			contentBase: path.join(process.cwd(), '/dist'),
-		},
-		plugins,
-		devtool: 'source-map',
-	};
-};
+module.exports = () => ({
+	mode: 'development',
+	watchOptions: {
+		aggregateTimeout: 300,
+		poll: 1000,
+	},
+	devServer: {
+		port: 3000,
+		stats: 'errors-only',
+		historyApiFallback: true,
+		contentBase: path.join(process.cwd(), '/dist'),
+	},
+	devtool: 'eval',
```

Comment:
why is it now set to eval?

Reply:
Do you prefer cheap-module-eval-source-map ? (https://webpack.js.org/configuration/devtool/ )

The old one was source map that is not dev friendly in build/rebuild time

Reply:
`cheap-module-source-map` for dev and `source-map` for th rest, yes.